resources/core/values.yaml

```diff
@@ -24,65 +24,65 @@ global:
     secretName: etcd-backup-abs-credentials
   acceptance_tests:
     dir:
-    version: 1.1.0-rc1
+    version: 1.1.0-rc2
   ui_acceptance_tests:
     dir:
     version: "83aa639b"
```

Comment:
is this on purpose or should it be `1.1.0-rc2`?

Reply:
It is on purpose, `ui_acceptance_tests` is part of a console, which is not being released.